MoEngagePluginBase/MoEngageInitConfig.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  MoEngageInitConfig.swift
+//  MoEngagePluginBase
+//
+//  Created by UdayKiran Naik on 02/07/24.
+//
+
+import Foundation
+
+public class MoEngageInitConfig {
```

Comment:
From where this class is created ?

Reply:
This is created in `MoECapacitorCorePlugin` class.

Reply:
This class need not be public.

Reply:
Since we are converting `JSObject` to `MoEngageInitConfig` in `MoECapacitorPlugin` it needs to be public otherwise we cannot create the `MoEngageInitConfig` object.